packages/smooth_app/lib/views/bottom_sheet_views/user_contribution_view.dart

```diff
@@ -60,13 +87,15 @@ class UserContributionView extends StatelessWidget {
     return showDialog<void>(
       context: context,
       builder: (BuildContext context) {
+        final AppLocalizations appLocalizations = AppLocalizations.of(context)!;
         return SmoothAlertDialog(
-          close: true,
-          title: AppLocalizations.of(context)!.contribute_improve_header,
+          close: false,
+          height: MediaQuery.of(context).size.height * 0.3,
```

Comment:
* 0.3 looks a bit small. I don't say that it's not right but is it the way you wanted